scripts/lib/browser-sdk-version.js

```diff
@@ -0,0 +1,5 @@
+const lernaJson = require('../../lerna.json')
+
+module.exports = {
+  getBrowserSdkVersion: lernaJson.version,
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** since we directly expose the value, it is not really a getter anymore ‏
```suggestion
  browserSdkVersion: lernaJson.version,
```